### PR TITLE
Migrate changeCSS(sideText) to CSS variable --sidebartext in utils/utils.jsx

### PR DESCRIPTION
**Pull request by KuSh**
#### Summary

Migrate changeCSS(sideText) to CSS variable --sidebartext in utils/utils.jsx

#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/15828
Fixes https://github.com/mattermost/mattermost-server/issues/15829
JIRA ticket: https://mattermost.atlassian.net/browse/MM-29389
JIRA ticket: https://mattermost.atlassian.net/browse/MM-29390

### CODE REVIEW

**Comment by deanwhillier** [comment]
QA review will be done on the final PR that merges all changes to this campaign branch into master.

**Review by hmhealey — APPROVED** [review]
Looks good! For context, the `.ps-` CSS classes were from the old [perfect-scrollbar](https://github.com/mdbootstrap/perfect-scrollbar) library that we used to use to style the scroll bars. We removed that a while ago though

**Comment by KuSh** [comment]
> Looks good! For context, the `.ps-` CSS classes were from the old [perfect-scrollbar](https://github.com/mdbootstrap/perfect-scrollbar) library that we used to use to style the scroll bars. We removed that a while ago though

Thanks @hmhealey I did explain that in the associated commit but forgot to report it on PR, thanks for that explanation 👍

**Comment by deanwhillier** [comment]
Will be QA'd when merged from the campaign branch into master.

**Review by asaadmahmood — APPROVED** [review]

File: components/user_settings/display/user_settings_theme/user_settings_theme.jsx [hunk 1]
```diff
@@ -81,10 +81,6 @@ export default class ThemeSetting extends React.PureComponent {
         };
     }
 
-    scrollToTop() {
-        $('.ps-container.modal-body').scrollTop(0);
```
[the remark this answers is not in the document]
Comment (deanwhillier) [reply components/user_settings/display/user_settings_theme/user_settings_theme.jsx:85]: Details were included in the commit: https://github.com/mattermost/mattermost-webapp/pull/6697/commits/a39f5d997b08c0447656e7c1b8932a3e66b78b2e

**Pull request merged**